realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -190,6 +192,7 @@ public static synchronized void init(Context context) {
             defaultConfiguration = new RealmConfiguration.Builder(context).build();
             ObjectServerFacade.getSyncFacadeIfPossible().init(context);
             BaseRealm.applicationContext = context.getApplicationContext();
+            SharedRealm.initialize(new File(context.getFilesDir(), ".realmNamedPipes"));
```

Comment:
We are using `.` as the separator in core, eg.: `default.management` and also, `temp` is more generic than pipes in case we need some other temp files in the future.
so what about `.realm.temp`?